product_assortment/models/ir_filters.py

```diff
@@ -73,13 +79,11 @@ def _get_action_domain(self, action_id=None):
     @api.multi
     def show_products(self):
         self.ensure_one()
-        return {
-            'type': 'ir.actions.act_window',
-            'name': _("Products"),
-            'res_model': 'product.product',
-            'domain': self._get_eval_domain(),
-            'view_type': 'form',
-            'view_mode': 'tree, form',
-            'context': self.env.context,
-            'target': 'current',
-        }
+        products = self.env['product.product'].search(self._get_eval_domain())
```

Comment:
This change is not wanted as that will generates two queries, the second one here under with ids will be slower, so put just the eval_domain in 'domain'